pkg/controller/daemon/daemon_controller.go

```diff
@@ -1147,6 +1147,7 @@ func (dsc *DaemonSetsController) updateDaemonSetStatus(ctx context.Context, ds *
 			}
 		} else {
 			if scheduled {
+				klog.V(2).InfoS("Found misscheduled daemon set pod", "daemonset", klog.KObj(ds), "node", node.Name)
```

Comment:
```suggestion
				klog.V(2).InfoS("Found misscheduled daemonset pod", "daemonset", klog.KObj(ds), "node", node.Name)
```

Reply:
Moved to `daemonset` with 867538b697a -> 770829aca30, but it diverges from the bulk of the current logging in this file:

```console
$ git --no-pager grep 'klog[.].*"[^"]*daemon *set[^"]*"' origin/master -- pkg/controller/daemon/daemon_controller.go | grep -o 'daemon *set' | sort | uniq -c
     12 daemon set
      2 daemonset
```

Did you want me to tack on a commit porting the existing log calls in this file while I'm touching things?

Reply:
let's leave it as "daemon set". However, that means that we should use "daemonSet" for the key. There are no precedents on this, from what I saw.

Reply:
Moved to `daemon set` and `daemonSet` with 770829aca30 -> 39be6f85acf.